app/models/concerns/with_qualifications.rb

```diff
@@ -39,7 +39,7 @@ module WithQualifications
     # subjects this course was tagged to.
     #
     # Defined here: https://github.com/DFE-Digital/manage-courses-api/blob/master/src/ManageCourses.Domain/Models/CourseQualification.cs
-    enum qualification: %i[qts pgce_with_qts pgde_with_qts pgce pgde]
+    enum qualification: %i[pgce_with_qts qts pgde_with_qts pgce pgde]
```

Comment:
before
![image](https://user-images.githubusercontent.com/470137/70140050-c6cfe500-168b-11ea-9bfc-bf89f317be91.png)

after
![image](https://user-images.githubusercontent.com/470137/70140077-d3ecd400-168b-11ea-948b-5de301e985a6.png)

You want something like what was describe here  https://github.com/DFE-Digital/manage-courses-backend/pull/1026#pullrequestreview-326432116


Can we leave a comment here as well so someone knows why it was hardcoded like that.